framework/vendor/interfaces/Emojis.java

```diff
@@ -13,5 +13,39 @@
 	String EMOJI_CHECKMARK_BOX = ":white_check_mark:";
 	String EMOJI_STOPWATCH = ":stopwatch:";
 	String EMOJI_ALARM_CLOCK = ":alarm_clock:";
-	
+	String EMOJI_TIRED_FACE = ":tired_face:";
```

Comment:
I'm also seeing many `EMOJI_[...]_FACE`, we should name those `EMOJI_FACE_[...]` instead so it will be easier to find the faces emojis in an editor that autocompletes name variables.

It's a little detail but it makes fast message writing a little cleaner to do, especially with the amount of emojis there is in Discord!